packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -37,22 +38,27 @@ export default {
     ],
   },
   create(context) {
+    const inputOptions = context.options && context.options[0];
```

Comment:
should we add .length check? Is there a chance of options being an empty array?
```
const inputOptions = context.options && context.option.length && context.options[0];
```

Reply:
@rheh I don't think this is necessary. The first element of an empty array is undefined.